src/components/pages/Analytics/TechStackGraph.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import axios from 'axios';
+import embed from 'vega-embed';
+
+export default function TechStackGraph() {
+  let url =
+    'http://underdog-devs-ds-a-dev.us-east-1.elasticbeanstalk.com/graph/tech-stack-by-role';
```

Comment:
here, we have deviated away from industry and our product's standards by hardcoding our API URL into the document. Let's move the API out into an `env var` and use the endpoints dynamically when firing of network requests. 

```
let url =`${process.env.API_URL}/graph/tech-stack-by-role` 
```

This will allow us to stay uniform and will protect our API URL's a little bit more, understanding that they can be accessed by peeking the network.

Reply:
Thank you for the guidance! Implemented API url into environment variable